content/4_Gold/DSU.mdx

```diff
@@ -172,20 +178,46 @@ class DisjointSets:
 		y_root = self.find(y)
 		if x_root == y_root:
 			return False
-		
-		if self.sizes[x_root] < self.sizes(y_root):
+
+		if self.sizes[x_root] < self.sizes[y_root]:
 			x_root, y_root = y_root, x_root
 		self.parents[y_root] = x_root
 		self.sizes[x_root] += self.sizes[y_root]
 		return True
+
+	# returns whether two nodes are in the same connected component
+	def connected(self, x: int, y: int) -> bool:
+		return self.find(x) == self.find(y)
 ```
 
 </PySection>
 </LanguageSection>
 
-## Focus Problem Solution
+## Solution - Focus Problem
 
-<FocusProblem problem="sample" />
+### Without Union Find (Floodfill)
```

Comment:
can you remove this part? I don't see a reason to add 1) a solution that doesn't use union find or 2) a solution that's too slow.

Reply:
I brought up a slower solution to show the main way we would solve it if union find didn't exist, and since it's pretty slow it shows why we have a motivation to actually use union find (kind of like with recursion being exponential time and dp reducing it to polynomial). But if you think that it's still not necessary, I'll be happy to take it out.

Reply:
I could also change it from a full on solution to a brief reference of the floodfill module

Reply:
> I could also change it from a full on solution to a brief reference of the floodfill module

👍